internal/adapters/events/forwarder_client.go

```diff
@@ -54,7 +54,7 @@ type ForwarderClient struct {
 
 // NewForwarderClient instantiate a new grpc forwarder client.
 func NewForwarderClient() *ForwarderClient {
-	cache := cache.New(24*time.Hour, 25*time.Hour)
+	cache := cache.New(24*time.Hour, 0)
```

Comment:
Why `0`?

Reply:
0 disables the janitor go routine that runs the cache evictions.
Since we are doing this manually, we don't need to use this